File: swagger/models/get_enrollments_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .class_schedule import ClassSchedule

class GetEnrollmentsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		enrollments
		Contains information about the enrollments.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Enrollments': ['enrollments', ClassSchedule],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'enrollments': 'Enrollments',
		}

	pagination_response = None
	enrollments = None

File: swagger/Models/GetEnrollmentsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetEnrollmentsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the enrollments.
	 * 
	 * @var ClassSchedule[] 
	 */
	public $enrollments;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Enrollments' => ['enrollments', 'array', ClassSchedule::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'enrollments' => ['Enrollments', 'array', ClassSchedule::class],
			];
	}
}
